components/Transactions.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import {
+  Card,
+  ContentContainer,
+  IconContainer,
+  MainText,
+  Overline
+} from '../styles/transactions';
+
+/**
+ * @prop
+ * */
+
+function Transactions(props) {
+  const { date, storeName, points } = props;
+  return (
+    <TouchableOpacity>
+      <Card>
+        <IconContainer />
+        <ContentContainer>
+          <Overline>
+            {date.toLocaleDateString()} @ {storeName}
```

Comment:
I added a 'Lookup' field in Airtable so that you could get 'storeName' back as expected, instead of a RECORD_ID (getting the value from a linked record field will always give you a record_id regardless of the primary key displayed in Airtable; so even though 'Store' shows the store names, when you get that value in code it's a record ID)

Reply:
I also changed all instances to be 'storeName' instead of 'storeId' for clarity

Reply:
Thats actually so much more efficient omg

Reply:
Airtable magic (^: woo